app/client/experiments/tests/example-test.ts

```diff
@@ -6,8 +6,8 @@ export const exampleTest: ABTest = {
 	expiry: '2020-12-01', // Remember that the server side test expiry can be different
 	author: 'anemailaddress@theguardian.com',
 	description: 'This Test',
-	audience: 0.0001, // 0.01% (1 is 100%)
-	audienceOffset: 0.5, // 50% (1 is 100%). Prevent overlapping with other tests.
+	audience: 1, // 0.01% (1 is 100%)
+	audienceOffset: 0, // 50% (1 is 100%). Prevent overlapping with other tests.
```

Comment:
should this be `0.5`?

Reply:
Not this one again 😅  But yes, either the values of the comments are incorrect and need changing. Maybe one to discuss when we go through this later as I'm not sure what we want these to be?